Source/WebKit/UIProcess/Cocoa/SystemPreviewControllerCocoa.mm

```diff
@@ -336,7 +340,8 @@ - (void)dataTask:(_WKDataTask *)dataTask didReceiveData:(NSData *)data
 - (void)dataTask:(_WKDataTask *)dataTask didCompleteWithError:(NSError *)error
 {
     if (error) {
-        FileSystem::closeFile(_fileHandle);
+        if (_started)
+            FileSystem::closeFile(_fileHandle);
```

Comment:
should we set `_started` back to false here and on line 354 or does that not matter?